### PR TITLE
supersedes #137: ZDI assigns the following CVEs:

### PR DESCRIPTION
**Pull request by kernelsmith**
M  2017/16xxx/CVE-2017-16590.json
M  2017/16xxx/CVE-2017-16591.json
M  2017/16xxx/CVE-2017-16592.json
M  2017/16xxx/CVE-2017-16593.json
M  2017/16xxx/CVE-2017-16594.json
M  2017/16xxx/CVE-2017-16595.json
M  2017/16xxx/CVE-2017-16596.json
M  2017/16xxx/CVE-2017-16597.json
M  2017/16xxx/CVE-2017-16598.json
M  2017/16xxx/CVE-2017-16599.json
M  2017/16xxx/CVE-2017-16600.json
M  2017/16xxx/CVE-2017-16601.json
M  2017/16xxx/CVE-2017-16602.json
M  2017/16xxx/CVE-2017-16603.json
M  2017/16xxx/CVE-2017-16604.json
M  2017/16xxx/CVE-2017-16605.json
M  2017/16xxx/CVE-2017-16606.json
M  2017/16xxx/CVE-2017-16607.json
M  2017/16xxx/CVE-2017-16608.json
M  2017/16xxx/CVE-2017-16609.json
M  2017/16xxx/CVE-2017-16610.json
M  2017/17xxx/CVE-2017-17406.json
M  2017/17xxx/CVE-2017-17407.json

### CODE REVIEW

File: 2017/16xxx/CVE-2017-16592.json [hunk 1]
```diff
@@ -11,7 +34,26 @@
       "description_data" : [
          {
             "lang" : "eng",
-            "value" : "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+            "value" : "This vulnerability allows remote attackers to disclose sensitive information on vulnerable installations of NetGain Systems Enterprise Manager 7.2.730 build 1034. Although authentication is required to exploit this vulnerability, the existing authentication mechanism can be bypassed. The specific flaw exists within the org.apache.jsp.u.jsp.restore.download_005fdo_jsp servlet, which listens on TCP port 8081 by default. When parsing the filename parameter, the process does not properly validate a user-supplied path prior to using it in file operations. An attacker can leverage this in conjunction with other vulnerabilities to execute code in the context of Administrator.  Was ZDI-CAN-5103."
```
Comment (cve-team) [inline 2017/16xxx/CVE-2017-16592.json]: What is the name of the affected component for CVE-2017-16592? org.apache.jsp.u.jsp.restore.download_005fdo_jsp is used in the descriptions for both CVE-2017-16592 and CVE-2017-16591. And while it does appear in the text in ZDI-17-957, the title has common.download_jsp.

Comment (kernelsmith) [reply 2017/16xxx/CVE-2017-16592.json]: addressed with commit 562b5037fe2870a483ece55c71a98b283e126e48

File: 2017/16xxx/CVE-2017-16592.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
...
+            "value" : "This vulnerability allows remote attackers to disclose sensitive information on vulnerable installations of NetGain Systems Enterprise Manager 7.2.730 build 1034. Although authentication is required to exploit this vulnerability, the existing authentication mechanism can be bypassed. The specific flaw exists within the common.download_jsp servlet, which listens on TCP port 8081 by default. When parsing the filename parameter, the process does not properly validate a user-supplied path prior to using it in file operations. An attacker can leverage this in conjunction with other vulnerabilities to execute code in the context of Administrator.  Was ZDI-CAN-5103."
```
Comment (cve-team) [inline 2017/16xxx/CVE-2017-16592.json]: Are you planning to update the name in the vulnerability details section of ZDI-17-957 too?

George

Comment (kernelsmith) [reply 2017/16xxx/CVE-2017-16592.json]: we are.  awaiting review by program management on that change (to our website)

**Pull request merged**